controllers/configurationpolicy_controller.go

```diff
@@ -1459,6 +1459,19 @@ func (r *ConfigurationPolicyReconciler) handleSingleObj(
 	if !exists && obj.shouldExist {
 		// it is a musthave and it does not exist, so it must be created
 		if strings.EqualFold(string(remediation), string(policyv1.Enforce)) {
+			// object is missing, so send noncompliant event
+			updateParent := createStatus("", obj.gvr.Resource, compliantObject, obj.namespaced, obj.policy,
```

Comment:
After thinking about this, I think createStatus needs to be able to append two statuses so that it just updates the `lastTransition` field on duplicates rather than always appending in the event an object is constantly updated each evaluation interval.